tests/OmniSharp.Tests/FixUsingsFacts.cs

```diff
@@ -0,0 +1,431 @@
+    {
+        public void method1()
+        {
+            List<string> first = new List<string>();
+            var testing = first.Where(s => s == ""abc"");
+        }
+    }
+}";
+
+            await AssertBufferContents(fileContents, expectedFileContents);
+        }
+
+        [Fact]
+        public async Task FixUsings_RemoveDuplicateUsing()
+        {
+            const string fileContents = @"using System;
+using System;
+
+namespace OmniSharp
+{
+    public class class1 
+    {
+        public void method1()
+        {
+            Console.WriteLine(""test"");
+        }
+    }
+}";
+
+            const string expectedFileContents = @"using System;
+
+namespace OmniSharp
+{
+    public class class1 
+    {
+        public void method1()
+        {
+            Console.WriteLine(""test"");
+        }
+    }
+}";
+            await AssertBufferContents(fileContents, expectedFileContents);
+        }
+
+        [Fact]
+        public async Task FixUsings_RemoveUnusedUsing()
+        {
+            const string fileContents = @"using System;
+using System.Linq;
+
+namespace OmniSharp
+{
+    public class class1 
+    {
+        public void method1()
+        {
+            Console.WriteLine(""test"");
+        }
+    }
+}";
+
+            const string expectedFileContents = @"using System;
+
+namespace OmniSharp
+{
+    public class class1 
+    {
+        public void method1()
+        {
+            Console.WriteLine(""test"");
+        }
+    }
+}";
+            await AssertBufferContents(fileContents, expectedFileContents);
+        }
+
+        private async Task AssertBufferContents(string fileContents, string expectedFileContents)
+        {
+            var response = await RunFixUsings(fileContents);
+            Assert.Equal(response.Buffer, expectedFileContents);
+        }
+
+        private async Task AssertUnresolvedReferences(string fileContents, List<QuickFix> expectedUnresolved)
+        {
+            var response = await RunFixUsings(fileContents);
+            var qfList = response.AmbiguousResults.ToList();
+            Assert.Equal(qfList.Count(), expectedUnresolved.Count());
+            var i = 0;
+            foreach (var expectedQuickFix in expectedUnresolved)
+            {
+                Assert.Equal(qfList[i].Line, expectedQuickFix.Line);
+                Assert.Equal(qfList[i].Column, expectedQuickFix.Column);
+                Assert.Equal(qfList[i].FileName, expectedQuickFix.FileName);
+                Assert.Equal(qfList[i].Text, expectedQuickFix.Text);
+                i++;
+            }
+        }
+
+        private async Task<FixUsingsResponse> RunFixUsings(string fileContents)
+        {
+            var workspace = TestHelpers.CreateSimpleWorkspace(fileContents, fileName);
+
+            var fakeOptions = new FakeOmniSharpOptions();
+            fakeOptions.Options = new OmniSharpOptions();
+            fakeOptions.Options.FormattingOptions = new FormattingOptions() {NewLine = "\r\n"};
```

Comment:
Pretty sure that the line ending should be `\n` here. Git converts the files to \n on commit. I think this will make the the tests pass in appveyor too.